src/__tests__/01a-price.ts

```diff
@@ -107,17 +102,17 @@ beforeAll(async () => {
 
 afterAll(async () => {
   return await mongoose.connection.close()
-});
+})
 
 it('test updating price', async () => {
   await price.update()
   // test it doesn't throw an error
 })
 
-it('test fetching last 24 hours', async () => {
-  const priceHistory = await price.lastCached()
-  // expect(priceHistory.length).toBe(25)
-})
+// it('test fetching last 24 hours', async () => {
```

Comment:
I guess this test was not very useful as is but why commenting it?

Reply:
Hoping someone will revive it one day :)